src/semeio/fmudesign/design_distributions.py

```diff
@@ -166,33 +183,36 @@ def draw_values_normal(dist_parameters, numreals, rng, normalscoresamples=None):
     mean = float(dist_parameters[0])
     stddev = float(dist_parameters[1])
 
-    if len(dist_parameters) == 2:
+    if len(dist_parameters) == 2:  # normal
         if normalscoresamples is not None:
-            return scipy.stats.norm.ppf(
+            values = scipy.stats.norm.ppf(
```

Comment:
isn't this equal to `values = mean +  normalscoresamples * stddev`?

Reply:
Yes I think so. Will change.